test/Test.hs

```diff
@@ -1,6 +1,44 @@
+import Devtools.Brittany (getBrittanyConfig)
+import Devtools.Stylish (getStylishConfig)
 import MPrelude
+import Test.Tasty
 
 import qualified Devtools
 
 main :: IO ()
-main = Devtools.main Devtools.defaultConfig
+main = do
+  stylishConfig <- getStylishConfig empty
+
+  brittanyConfig <- getBrittanyConfig empty
+
+  let ormoluFormatter =
+        Devtools.FormatterOptions
+          { formatter = Devtools.Ormolu
+          , files = ["test/Examples/Ormolu.hs"]
+          }
+
+  let stylishFormatter =
+        Devtools.FormatterOptions
+          { formatter = Devtools.Stylish stylishConfig
+          , files = ["test/Examples/Stylish.hs"]
+          }
+
+  let brittanyFormatter =
+        Devtools.FormatterOptions
+          { formatter = Devtools.Brittany brittanyConfig
+          , files = ["test/Examples/Brittany.hs"]
```

Comment:
The `=` should be aligned with the line above? (Same with the other two groups above)

Reply:
Yeah, good catch.